twarep/electrifly
# UI Analytics Welcome

## Checklist

## What I Did
Added bootstrap.css
Added a card as the welcome header

## How I Did it?
Added a bootstrap.css file and changed the colours under card-header. Then this was called in app.py and the colour should have been changed to blue. We also used a library called shiny.experimental which was used to make the cards.

## How to Test?
Run the app.py file and see the welcome message.
<img width="593" alt="image" src="https://github.com/twarep/electrifly/assets/64234627/1eca6107-b06d-4b5f-903a-3e3c6f5da7b8">

### Relevant Links
- https://shiny.posit.co/py/api/ExCard.html
- https://bootswatch.com/zephyr/
